eventing/samples/github-source/README.md

```diff
@@ -21,13 +21,31 @@ You will need:
   Eventing](https://github.com/knative/docs/tree/master/eventing). Those
   instructions also install the default eventing sources, including
   the `GitHubSource` we'll use.
-1. Create a `Channel`. You can use your own `Channel` or use the
-   provided sample, which creates a channel called `githubchannel`. If
-   you use your own `Channel` with a different name, then you will
-   need to alter other commands later.
+
+### Create a Knative Service
+
+To verify the `GitHubSource` is working, we will create a simple Knative
+`Service` that dumps incoming messages to its log.  Let's create that first.
+
+```yaml
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: github-message-dumper
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper
+```
+
+This can be done from `service.yaml`:
```

Comment:
"This can be done from `service.yaml`:" ---> "Enter the following command to create the service from `service.yaml`:"